### PR TITLE
docs(docs_data_analysis/index): add installation section

### PR DESCRIPTION
**Pull request by Yalyalieva**
installation section + small fixes in documentation text

### CODE REVIEW

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md [hunk 1]
```diff
@@ -9,6 +9,18 @@ This package is a comprehensive solution for data query, analysis and visualizat
 
 The package is devided into three modules: [Data access](data_access/index.md), [Error analysis](error_analysis/index.md) and [Validation](validation/index.md).
```
Comment (mariaBer) [inline docs_data_analysis_versioned_docs/version-v0.7.17/index.md:10]: remove

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md [hunk 2]
```diff
@@ -9,6 +9,18 @@ This package is a comprehensive solution for data query, analysis and visualizat
```
Comment (mariaBer) [inline docs_data_analysis_versioned_docs/version-v0.7.17/index.md:9]: This package installed as part of  CITROS CLI.
To use the package you can either use it in CITROS.io or locally after installing and logging in with [CITROS CLI](/docs_cli/overview/cli_install).

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md [hunk 3]
```diff
@@ -9,6 +9,18 @@ This package is a comprehensive solution for data query, analysis and visualizat
 
 The package is devided into three modules: [Data access](data_access/index.md), [Error analysis](error_analysis/index.md) and [Validation](validation/index.md).
 
+## Installation
```
Comment (mariaBer) [inline docs_data_analysis_versioned_docs/version-v0.7.17/index.md:12]: Remove this part

**Review by mariaBer — CHANGES_REQUESTED** [review]

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 1 above)
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:10]: Why information about modules should be removed?

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 3 above)
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:12]: You mean to remove the title or the whole section?

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 2 above)
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:9]: You want me to right it on the 9th line? It is important to stress that you not just working locally, but you still work with data from the cloud. In other tutorial by 'locally' means that you run simulations and got data locally, it is not the case

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 3 above)
Comment (mariaBer) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:12]: The text I added on top is a substitute for this part

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 1 above)
Comment (mariaBer) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:10]: The TOC reflects the three modules

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 2 above)
Comment (mariaBer) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:9]: Data analysis is a python package installed as part of CITROS CLI.
This gives you the ability to query and analyze data generated by CITROS. 
The package can be used either from citros.io python notebook located under each [repository notebook](/docs/repos/repository/repo_sc_code.md#directory-notebooks) or from local kernel python environment such as jupyter, VSC, etc.

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 3 above)
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:12]: But it is essential to write how to update the package. If the new version is released it is not updated, it is even not updated if you update citros. You can update it only if you remove citros and install it again or, what looks much better, when you update the package itself

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 2 above)
Comment (Yalyalieva) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:9]: I still would like to add that it works with cloud data only

File: docs_data_analysis_versioned_docs/version-v0.7.17/index.md (hunk 3 above)
Comment (mariaBer) [reply docs_data_analysis_versioned_docs/version-v0.7.17/index.md:12]: Understood, however, no need to add it as an explanation. Its in fact an issue of CITROS CLI and should be fixed

**Comment by mariaBer** [comment]
@Yalyalieva I'm running the branch and dont see the changes

**Comment by Yalyalieva** [comment]
@mariaBer 
Sorry, forgot to transfer changes to the versioned document, they were only in 'next'

File: docs_data_analysis/index.md [hunk 4]
```diff
@@ -7,19 +7,9 @@ hide_title: true
...
-If you wish to work with cloud data remotely, for example using Visual Studio Code, the installation of [CITROS CLI](https://citros.io/doc/docs_cli/overview/cli_install) is required. The **citros_data_analysis** package will be installed alongside it. To use **citros_data_analysis** package you need to be [logged in](https://citros.io/doc/docs_cli/overview/cli_quickstart#run-remotely-on-the-cloud) in your CITROS account.
-
-To update **citros_data_analysis** package without reinstalling CITROS CLI, use pip:
-
-```bash
-$ pip install -U citros-data-analysis
-```
+**citros_data_analysis** is a python package installed both in [web](https://citros.io/) and as part of CITROS CLI.
+This gives you the ability to query and analyze data generated by running simulation in CITROS cloud.
+The package can be used either from [citros.io](https://citros.io/) python notebook located under each repository notebook or from local kernel python environment such as jupyter, VSC, etc. Iin the latter case to use **citros_data_analysis** package you need to be logged in your CITROS account using [CITROS CLI](https://citros.io/doc/docs_cli/overview/cli_quickstart#run-remotely-on-the-cloud).
```
Comment (mariaBer) [inline docs_data_analysis/index.md:12]: **citros_data_analysis** is a Python package that enables you to query and analyze data generated by running simulations in the CITROS cloud. This package can be used either from the citros.io Python notebook located under each [repository's notebook](docs/repos/repository/repo_sc_code#directory-notebooks) or from a local kernel Python environment such as Jupyter, VSC, etc., after installing the [CITROS](https://pypi.org/project/citros/) package and logging in.

The text above approved by Vova, if you have any questions/comment, discuss them with him

**Pull request merged**